sled-agent/src/sled_agent.rs

```diff
@@ -424,14 +444,34 @@ impl SledAgent {
     }
 
     async fn ensure_scrimlet_services_active(&self, log: &Logger) {
-        // TODO(https://github.com/oxidecomputer/omicron/issues/823): Launch the switch zone, with
-        // Dendrite, MGS, and any other services we want to enable.
-        warn!(log, "Activating scrimlet services not yet implemented");
+        info!(log, "Ensuring scrimlet services (enabling services)");
+
+        let services = match self.inner.config.stub_scrimlet {
```

Comment:
I suspect we'll need additional configuration options to detect whether or not to launch SoftNPU here too.